src/uu/dd/Cargo.toml

```
@@ -21,7 +21,7 @@ path = "src/dd.rs"
 clap = { workspace = true }
 gcd = { workspace = true }
 libc = { workspace = true }
-uucore = { workspace = true, features = ["format", "parser", "quoting-style"] }
+uucore = { workspace = true, features = ["format", "parser", "quoting-style", "fs"] }
```

This should make the `Style/toml` job in the CI pass:
```suggestion
uucore = { workspace = true, features = [
  "format",
  "parser",
  "quoting-style",
  "fs",
] }
```

---

Thx, I just run the formatter for me locally via "Event Better TOML" (a vscode extension) but `npx --yes @taplo/cli fmt src\uu\dd\Cargo.toml` did not work for me for some reason. I'm on Windows.